.github/workflows/run-fullnode-sync.yaml

```diff
@@ -0,0 +1,79 @@
+# This workflow is a simple wrapper around the fullnode-sync github
```

Comment:
nice! you might be able to dedup this further even. Something I'm playing with and considering for our GHA is to define jobs like this in a single file like `run-fullnode-sync-all.yaml` that invokes the reusable workflow a bunch of times: like here: https://github.com/aptos-labs/aptos-core/pull/6435/files#diff-3efa580731ca445ce0e8063eabb4668a669b537d270a4d864a2154129c1ea43e

You could also use such a design to more clearly define different workflow run cadences. 

This cleans up workflows that need to be invoked with a bunch of different options, like:
* continuous forge
* docker build
* replay-verify and module-verify (against different networks)

Reply:
Nice! 😄 I think I'm gonna defer this to a post V1 follow up/clean up (likely next week). Agree though. Would be awesome to have everything in one location 🚀